clients/csharp-dotnet-core/voice-assistant-test/tool/TurnResult.cs

```diff
@@ -42,11 +43,17 @@ public TurnResult(Turn turn)
         public string ActualRecognizedText { get; set; }
 
         /// <summary>
-        /// Gets or sets the Actual TTS Audio Reponse Duratio (in milliseconds).
+        /// Gets or sets the Actual TTS Audio Response Duration (in milliseconds).
         /// </summary>
         [JsonProperty(Order = 7, DefaultValueHandling = DefaultValueHandling.Ignore)]
         public List<int> ActualTTSAudioResponseDurations { get; set; }
 
+        /// <summary>
+        /// Gets or sets the actual user perceived latency (in milliseconds).
+        /// </summary>
+        [JsonProperty(Order = 8, DefaultValueHandling = DefaultValueHandling.Ignore)]
+        public int ActualUserPerceivedLatency { get; set; }
```

Comment:
Is there an ExpectedUserPerceivedLatency? Every "ActualXxx" should have an "ExpectedXxx". He you still call it ExpectedResponseLatency

Reply:
will rename to ExpectedResponseLatency

Reply:
Done